tests/cases/fourslash/codeFixInferFromUsageMember2JS.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+// @checkJs: true
+// @noImplicitAny: true
+// @Filename: important.js
+
+/////** @typedef {{ [|p |]}} I */
+/////** @type {I} */
+////var i;
+////i.p = 0;
+
+
+verify.rangeAfterCodeFix("p: number", undefined, undefined, 1);

```

Comment:
The range includes a trailing space but the replacement text does not -- does this end up looking like `{{ p: number}}` instead of `{{ p: number }}`?

Reply:
No, I tested it manually. I think this is an oddity of the fourslash span code.